src/azul/plugins/__init__.py

```diff
@@ -471,13 +471,6 @@ def resolve_source(self, spec: str) -> SOURCE_REF:
         id = self.lookup_source_id(spec)
         return ref_cls(id=id, spec=spec)
 
-    def verify_source(self, ref: SOURCE_REF) -> None:
-        """
-        Verify that the source's ID matches that defined in the
-        repository for the source's spec.
```

Comment:
Since this is now a responsibility of `lookup_source` it should be stated explicitly in that method's docstring.

Reply:
@hannes-ucsc: "comment is invalid"